PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/container/utils/DockerUtil.java

```diff
@@ -50,6 +52,8 @@
 
 
 public class DockerUtil {
```

Comment:
why there are two same DockerUtil?

Reply:
they are for eclipse/intellij separately.
they should be merged and shared. there are many other such kind of duplicate classes in eclipse/intellij.
we can refactor in the future.

Reply:
ok